assignment-2/7s.md

```diff
@@ -1,2 +1,296 @@
+
+##### Structure
+
+> Structure is about how the organization is arranged or configured. Many organizations take a
+> functional structure, in which different branches of the organization specialize in particular
+> functions, such as manufacturing, sales, and service. Another common form is a divisional
+> structure, in which divisions are arrayed according to geographical regions, products, or some
+> other factor, and each division contains a full complement of functional elements (e.g., sales,
+> service, R&D).
+
+Spotify headquarters resides in London and its handles, research and development departments are in Stockholm.
+Like many other matrix management organizations, Spotify subdivides the work among people with similar skills, and they are pooled for work assignments, resulting in more than one manager with their own responsibilities and competencies. All this managers are from that point, reporting hierarchical button up to the CTO and finally to the CEO.
+Further literature does not specify aspect like functional elements (e.g., sales, service, R&D), and how they interact with eachother.
+
+##### Systems
+
+> Systems refers to the rules and regulations, standards, and processes for getting things done and managing the activity of the organization, both formal and informal.
+
+How things get done? 
+according to Spotify [12]: 
+- Spotify has a culture based on agile principles, that went down to Scum at early years, but some of the scrum practices were getting on the way so they decided to make them optional. Perhaps because they valorate more the agile principals than the scrum practices.
+- All engineerings happens in squads (preferably less than 8 members) , they try to keep those squads loosely coupled, tightly aligned. I could not find the reason of them using squads, but perhaps they do it attempting to minimizing dependencies.
+- They like cross-pollination instead of 'standardization', in order to spread knowledge, trends and tools between squads. It does not break the principal of 'isolated missions' because cross-pollination is used to spread the uses of tools like github for example. When there is not an standard for the use of tools, they could perhaps loose some consistency, anyway literature indicates that Spotify culture stands for flexibility and they found a happy balance.
+- Their code is shared on an internal open source model. Again this can be an attempt to avoid dependencies.
+- Squads make small and frequent releases as a routine, which is enable due decoupling their services.
+- Their uses release trains and feature toggles aiming get things into production early and often.
```

Comment:
I think it should be "They use release trains and feature toggles aiming get things into production early and often."